docs/user_guide/HOWTO/Kerberos.rst

```diff
@@ -96,19 +105,19 @@ Propagate the Keytabs
 ^^^^^^^^^^^^^^^^^^^^^
 
 Move all of the resulting keytab files SECURELY to
-`<environment_dir>/keydist/<fqdn>/keytabs` on the Puppet server as appropriate
-for each file.
+``/var/simp/environments/<client_environment>/site_files/krb5_files/keytabs/<fqdn>``
```

Comment:
I think this is /var/simp/environments/<client_environment>/site_files/krb5_files/**files**/keytabs/fqdn